### PR TITLE
Fix OpenIdConnect test

### PR DESCRIPTION
**Pull request by troydai**
https://github.com/aspnet/Security/pull/925

/cc @AndriySvyryd @Tratcher

### CODE REVIEW

File: test/E2ETests/Implementation/OpenIdConnectLoginScenarios.cs [hunk 1]
```diff
@@ -110,9 +110,12 @@ public partial class Validator
                 "https://login.windows.net/4afbc689-805b-48cf-a24c-d4aa3248a248/oauth2/logout",
                 response.Headers.Location.AbsoluteUri.Replace(response.Headers.Location.Query, string.Empty));
             queryItems = new QueryCollection(QueryHelpers.ParseQuery(response.Headers.Location.Query));
-            Assert.Equal<string>(_deploymentResult.ApplicationBaseUri, queryItems["post_logout_redirect_uri"]);
+            Assert.Equal<string>(_deploymentResult.ApplicationBaseUri + "signout-callback-oidc", queryItems["post_logout_redirect_uri"]);
```
Comment (Tratcher) [inline test/E2ETests/Implementation/OpenIdConnectLoginScenarios.cs]: No state?

Comment (Tratcher) [inline test/E2ETests/Implementation/OpenIdConnectLoginScenarios.cs]: or it just flows separately?

Comment (troydai) [inline test/E2ETests/Implementation/OpenIdConnectLoginScenarios.cs]: No, this is validating the "post logout uri" flowed among the query. There are two pairs here: 1) `post_logout_redirect_uri` 2) `state`

**Comment by Tratcher** [comment]
:shipit:

**Comment by troydai** [comment]
I will not wait for the CI since this is fixing a CI build break.

**Pull request merged**